src/users/dto/create-user.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateUserDto {
+  @ApiProperty({type:String})
+    id: string;
+
+    @ApiProperty({type:String , description:'first name'})
+    first_name: string;
+
+    @ApiProperty({type:String , description: 'last name'})
+    last_name: string;
+
+    @ApiProperty({type:String , description: 'password'})
```

Comment:
one more attribute that can be added here called `default`, this would give an initial value for each entity. we can add it for now, or we can generate some dummy data using `Faker` library for the next step. Adding this attribute or not is up to you :grin: